2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2066 @@
+> Finally, they release the first major version of the library, named
+> *hal9k* and publish it through the university library’s software
+> portal, where they get a DOI (Digital Object Identifier) for the
+> version as well as a concept DOI for any future versions of the
+> library. Working with the RSE, Kim has gained a good understanding of
+> some methods in software engineering, and she’s thrilled because this
+> also means she’ll be able to get a job with a local tech company once
+> her fixed-term contract has run out.  
+> Kim passes her PhD - of which *hal9k* is an important part - with
+> flying colours, and soon citations to her library start appearing in
+> the researchonomic literature. To Kim’s surprise, she also reads a
+> blog post about a citizen science maker project which has used *hal9k*
+> to process researchonomic data measured in a neighbourhood of her
+> hometown. She is invited to give a talk at the local office of Siren,
+> a global tech company, which look to adopt *hal9k*, and pay Kim
+> a generous speaker honorarium. So generous in fact, that Kim can pay a student
+> assistant for a full year from the money.
+
+Our credibility as researchers in society hinges on the notion of proper
+research conduct, also known as “good research practice”. The
+digitalization of research has introduced complex digital research
+outputs, such as software and data sets. Although first
+recommendations (Katerbow and Feulner 2018) and policies (Scheliga et
+al. 2019) exist, they are far from being widely adapted. It is still
+somewhat unclear how to translate good research practice into good research
+software practice, for example in terms of validity and reproducibility,
+but also pertaining to the responsible use of resources. The damage
+that failing to do is causing both to the progress of the research community
+and to the credibility of academic research in society is becoming increasingly clear
+with the growth of the replication crisis - while the lack of universally
+agreed-upon and supported good research software practice is not the main
+reason for that crisis, it clearly is a contributing factor.
+
+While it is obvious that software qualifies as a potentially re-usable
+digital artifact, the additional benefit of not just reproducing a given
+scenario, but transferring software use to new problems, domains, and/or
+applications, justifies developing research software with a long-term
+perspective as *sustainable research software*.
+
+In order to support research, a sustainable software must be
+correct (Hatton 2007; Chang et al. 2006; Matthews 2007), validatable,
+understandable, documented, publicly released, adequately published
+(i.e. in persistently identifiable form as software source code (Smith
+et al. 2016), and potentially in an additional paper which describes the
+software concept, design decisions and development rationale), actively
+maintained, and (re-)usable (Merali 2010; Barnes 2010; Tse 2010).
```

Comment:
Added the following paragraph after this line to resolve https://github.com/DE-RSE/positions/pull/1/files#r370522502:

```
We also argue that truly sustainable research software must be published under a Free and Open Source Software license, and follow an open development model, to (1) enable the validation of research results that have been produced using software, (2) enable the reproducibility of software-based research, (3) enable improvement and (re-) use of the software to support more and better research, and reduce resources to be spent on software development, (4) reduce legal issues (see section on legal issues below), (5) meet ethical obligations from public funding, and (6) open research software to the general public, i.e., the stakeholder group with arguably the greatest interest in furthering research knowledge and improving research for the benefit of all.
```

@bussec What do you think?

Reply:
@sdruskat Fine with me! Maybe include an additional point that focuses more from the open development process, e.g., "(X) facilitate the growth of a community of developers and users around the project".

Also, I agree with @knarrff as the first line is rather explicit in its wording and not everyone shares this level of enthusiasm for Free Software ;-)